### DISCUSSION TITLE
Sample Regions Demo throws exception when Content is added to ContentControl. Complains about ContentControl region adapter not there

### DISCUSSION THREAD

**Discussion by stevensrf11** [PrismLibrary/Prism#2387] [WPF]
Is anyone getting this issue with the latest sample of the PRISM.
I downloaded the Sample Prism source code.
In Sample 2 Regions demo I add some Content to the Content Control.
ContentControl prism:RegionManager.RegionName="ContentRegion" Content="Shell Content"
When I ran the demo in Visual Studio 2019 an exception was thrown complaining about ContentController not having a region adapter created for it.
The Prism Version is prism.unity 8.0.0.1909

![Untitled](https://user-images.githubusercontent.com/22349374/114402455-757e1280-9b69-11eb-9950-234fa5236be3.png)

[02-Regions.zip](https://github.com/PrismLibrary/Prism/files/6297048/02-Regions.zip)

**Comment by brianlagunas** [comment]
See #2389
